js/join.js

```diff
@@ -182,18 +182,33 @@ function arePersonalDetailsValid() {
     country.value.trim() != ''
   );
 }
-
 function introPageValidator() {
+  let countIntroduction = 0;
+  let textAreaIntroduction = introduction.value;
+  let textAreaSkills = skills.value;
+  let countSkills = 0;
+  let textAreaCollege = college.value;
+  let countCollege = 0;
+  let textAreaForFun = forFun.value;
+  let countForFun = 0;
+  let textAreafunFact = funFact.value;
+  let countfunFact = 0;
+  countIntroduction = countWords(textAreaIntroduction);
```

Comment:
want to know you are using countWords function here(i.e line- 196) and on line 446 also?
what is the difference between both??

Reply:
In this both function has same working, here, in 196 I'm using because when user fill correctly data .Then, next button enable .
And in 446, it check the data validation. If  the data is incorrect then red box which appear , and apart from that if data is correct then. Red-box shouldn't be appear on the textarea field.